versions.tf

```diff
@@ -11,16 +11,12 @@
 // limitations under the License.
 
 terraform {
-  required_version = ">= 1.5.0, <= 1.5.5"
+  required_version = "<= 1.5.5"
```

Comment:
I don't see a reason to remove this constraint, let's revert this line.

Reply:
fixed